src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Providers/IPsBackupProvider.cs

```diff
@@ -43,6 +44,9 @@ public interface IPsBackupProvider
 
         ProtectionPolicyResponse GetPolicy();
 
+        AzureRmRecoveryServicesSchedulePolicyBase GetDefaultSchedulePolicyObject();
+
+        AzureRmRecoveryServicesRetentionPolicyBase GetDefaultRetentionPolicyObject();
```

Comment:
Please make sure that these methods are implemented in all the providers. This would otherwise cause a build break.